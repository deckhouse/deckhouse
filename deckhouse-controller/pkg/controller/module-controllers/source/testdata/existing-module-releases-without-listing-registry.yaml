---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: 90f0955ee984feab5c50611987008def
  name: test-source-1
spec:
  registry:
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/deckhouse/modules
    scheme: HTTPS
status:
  message: ""
  modules:
    - name: parca
---
apiVersion: deckhouse.io/v1alpha2
kind: ModuleUpdatePolicy
metadata:
  name: test-alpha
spec:
  releaseChannel: Alpha
  update:
    mode: Manual
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: parca
properties:
  version: 1.4.1
  source: test-source-1
  availableSources:
    - test-source-1
status:
  phase: Available
  conditions:
    - type: EnabledByModuleConfig
      status: "True"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.1
spec:
  changelog:
    features:
      - Bump parca version
  moduleName: parca
  version: 1.4.1
  weight: 900
status:
  approved: false
  message: ""
  phase: Deployed
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.2
spec:
  changelog:
    features:
      - Bump parca version
  moduleName: parca
  version: 1.4.2
  weight: 900
status:
  approved: false
  message: ""
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.3
spec:
  changelog:
    features:
      - Bump parca version
  moduleName: parca
  version: 1.4.3
  weight: 900
status:
  approved: false
  message: ""
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
