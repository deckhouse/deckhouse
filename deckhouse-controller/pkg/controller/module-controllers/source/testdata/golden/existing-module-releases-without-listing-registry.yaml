---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: 90f0955ee984feab5c50611987008def
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/module-exists
  name: test-source-1
  resourceVersion: "1001"
spec:
  registry:
    ca: ""
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/deckhouse/modules
    scheme: HTTPS
status:
  message: ""
  modules:
  - checksum: 'sha256:'
    name: parca
    version: v1.4.4
  modulesCount: 1
  phase: Active
  syncTime: "2019-10-17T15:33:00Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.1
  resourceVersion: "999"
spec:
  changelog:
    features:
    - Bump parca version
  moduleName: parca
  version: 1.4.1
  weight: 900
status:
  approved: false
  message: ""
  phase: Deployed
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.2
  resourceVersion: "999"
spec:
  changelog:
    features:
    - Bump parca version
  moduleName: parca
  version: 1.4.2
  weight: 900
status:
  approved: false
  message: ""
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: foxtrot-alpha
    release-checksum: 98d00f741c99e06e6c6c4d18b763c550
    source: foxtrot
  name: parca-v1.4.3
  resourceVersion: "999"
spec:
  changelog:
    features:
    - Bump parca version
  moduleName: parca
  version: 1.4.3
  weight: 900
status:
  approved: false
  message: ""
  phase: Pending
  pullDuration: 0s
  size: 0
  transitionTime: "2024-05-03T20:55:49Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    modules.deckhouse.io/change-cause: check release (from deployed)
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: ""
    release-checksum: 1beb143dffb1b662137094e7faea1e17
    source: test-source-1
  name: parca-v1.4.4
  ownerReferences:
  - apiVersion: deckhouse.io/v1alpha1
    controller: true
    kind: ModuleSource
    name: test-source-1
    uid: ""
  resourceVersion: "1"
spec:
  moduleName: parca
  version: 1.4.4
  weight: 900
status:
  approved: false
  message: ""
  pullDuration: 0s
  size: 0
  transitionTime: null
